src/user/message.spec.ts

```diff
@@ -44,7 +44,7 @@ test('recalled()', async t => {
   const sandbox = sinon.createSandbox()
 
   const puppet = new PuppetMock()
-  const wechaty = new Wechaty({ puppet })
+  const wechaty = new Wechaty({ puppet: 'wechaty-puppet-mock' })
```

Comment:
`puppet` needs to be tested in the below assertions.

Please revert to make the unit test workable.